sdk/storage/azure-storage-blobs/inc/azure/storage/blobs.hpp

```diff
@@ -12,9 +12,13 @@
 #include "azure/storage/blobs/blob_client.hpp"
 #include "azure/storage/blobs/blob_container_client.hpp"
 #include "azure/storage/blobs/blob_lease_client.hpp"
+#include "azure/storage/blobs/blob_options.hpp"
+#include "azure/storage/blobs/blob_responses.hpp"
 #include "azure/storage/blobs/blob_sas_builder.hpp"
 #include "azure/storage/blobs/blob_service_client.hpp"
 #include "azure/storage/blobs/block_blob_client.hpp"
 #include "azure/storage/blobs/dll_import_export.hpp"
 #include "azure/storage/blobs/page_blob_client.hpp"
-#include "azure/storage/common/storage_exception.hpp"
+#include "azure/storage/blobs/rest_client.hpp"
+#include "azure/storage/blobs/rtti.hpp"
+#include <azure/storage/common/storage_exception.hpp>
```

Comment:
We divide include headers into groups in storage, following [this](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) rule. 

So please move this header to after `#pragma once` and make it a separate group.

Reply:
Done - here, and 3 other occurrences.